backend/curriculum_tracking/tests/test_UserStatsPerWeekSerializer.py

```diff
@@ -0,0 +1,234 @@
+
+    def test_create_four_reviews_cards_will_be_in_different_columns(self):
+
+        # Request for a review to happen on project_1, card_1
+        request_review_time = self.project_1.start_time + timedelta(1)
+        self.project_1.request_review(force_timestamp=request_review_time)
+        time_one = self.project_1.start_time - timedelta(days=6)
+        time_two = self.project_1.start_time + timedelta(days=4)
+        time_three = self.project_1.start_time + timedelta(days=3)
+
+        # Three reviews are made at different times, so we should have different cards in different columns depending
+        # on the outcome of the review
+        review_1 = RecruitProjectReviewFactory(
+            status=NOT_YET_COMPETENT,
+            recruit_project=self.project_1,
+            timestamp=time_one
+        )
+        review_1.timestamp = time_one
+        review_1.save()
+
+        review_2 = RecruitProjectReviewFactory(
+            status=COMPETENT,
+            recruit_project=self.project_1,
+        )
+        review_2.timestamp = time_two
+        review_2.save()
+
+        review_3 = RecruitProjectReviewFactory(
+            status=EXCELLENT,
+            recruit_project=self.project_1,
+        )
+        review_3.timestamp = time_three
+        review_3.save()
+
+        # review_1 had a status of NYC, so we should have at least one card in the 'REVIEW FEEDBACK' column
+        assert UserStatsPerWeekSerializer.get_cards_in_review_feedback_column(
+            UserStatsPerWeekSerializer, User=self.user
+        ) == 1
+
+        # review_2 had a status of COMPETENT, the reviewer is not trusted and therefore we should have zero cards in
+        # the completed column
+        assert UserStatsPerWeekSerializer.get_cards_in_completed_column(
+            UserStatsPerWeekSerializer, User=self.user
+        ) == 0
+
+        # review_3 had a status of Excellent, the reviewer is not trusted and therefore we should have zero cards in
+        # the completed column
+        assert UserStatsPerWeekSerializer.get_cards_in_completed_column(
+            UserStatsPerWeekSerializer, User=self.user
+        ) == 0
+
+        # In total we did start new cards within the last 7 days but we did not complete any cards in the last seven
+        # days
+        assert UserStatsPerWeekSerializer.get_cards_started_last_7_days(
+            UserStatsPerWeekSerializer, User=self.user
+        ) > 0
+        assert UserStatsPerWeekSerializer.get_cards_completed_last_7_days(
+            UserStatsPerWeekSerializer, User=self.user
+        ) == 0
+
+    def test_one_card_in_complete_column(self):
+
+        content_item_2 = ContentItemFactory(
+            content_type=ContentItem.TOPIC,
+            project_submission_type=ContentItem.REPOSITORY,
+        )
+        card_2 = AgileCardFactory(
+            content_item=content_item_2,
+            status=AgileCard.COMPLETE,
+        )
+        assignee__ = SocialProfileFactory().user
```

Comment:
Why `assignee__` instead of just `assignee`? What are you trying to tell us with this snew naming convention?